AquaNet/src/components/UserBox.svelte

```diff
@@ -347,8 +341,10 @@
       user = u;
       const card = user.cards.length > 0 ? user.cards[0].luid : "";
 
-      if (card) {
-        fetchData(card);
+      aimeId = card;
```

Comment:
Didn't know all the cards of the user were under the id of the ghost card. Modified the code so that it grabs that directly instead of asking the server.